internal/inspectimage/info_display.go

```diff
@@ -187,12 +187,18 @@ func convertToDisplay(proc launch.Process, isDefault bool) ProcessDisplay {
 	case false:
 		shell = "bash"
 	}
+	var argsToUse []string
+	if proc.Args != nil {
+		argsToUse = proc.Args
+	} else {
+		argsToUse = proc.Command.Entries[1:]
```

Comment:
Should we check the length here? To avoid index out of range?

Reply:
Thank you @natalieparellano 🙇 I've just added a length check.